### PR TITLE
Update bloom

### PR DESCRIPTION
**Pull request by LinKArftc**
Re: cocos-creator/3d-tasks#

Changelog:
 *

### CODE REVIEW

File: cocos/core/pipeline/common/bloom-stage.ts [hunk 1]
```diff
@@ -93,6 +110,8 @@ export class BloomStage extends RenderStage {
         if (!camera.window?.swapchain && !pipeline.macros.CC_PIPELINE_TYPE) {
             return;
         }
+        if (!pipeline.bloomEnabled) return;
```
Comment (2youyou2) [inline cocos/core/pipeline/common/bloom-stage.ts]: 现在 uiphase 在 lighting stage，得剔除掉 ui camera

Comment (LinKArftc) [reply cocos/core/pipeline/common/bloom-stage.ts]: 怎么区分 ui camera 呢？

**Review by pandamicro — APPROVED** [review]

File: cocos/core/pipeline/common/bloom-stage.ts (hunk 1 above)
Comment (LinKArftc) [reply cocos/core/pipeline/common/bloom-stage.ts]: Done

**Comment by pandamicro** [comment]
Native https://github.com/cocos-creator/engine-native/pull/3970

**Pull request merged**